### PR TITLE
feat: replace map with parallel-transform

### PR DESCRIPTION
**Pull request by tpluscode**
Not concurrent by default

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +34,7 @@
     "isstream": "^0.1.2",
     "mocha": "^7.0.0",
     "nyc": "^15.0.0",
-    "standard": "^14.3.1"
+    "standard": "^14.3.1",
+    "stream-array-es6": "^1.0.1"
```
Comment (bergos) [inline package.json]: Have you thought about using [into-stream](https://www.npmjs.com/package/into-stream)? It's more popular and has more releases and more recent releases, so hopefully, it went through more quality checks.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[*]
+indent_size = 2
+indent_style = space
```
Comment (bergos) [inline .editorconfig:3]: Not really part of the PR

**Review by bergos — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (tpluscode) [reply package.json]: Probably it somehow showed up higher in my search results. I can switch

File: .editorconfig (hunk 2 above)
Comment (tpluscode) [reply .editorconfig:3]: Do you really oppose adding this? Without editorconfig the default formatting was all wring in my IDE

**Review by bergos — APPROVED** [review]
